Sources/Remote Logging/Crash Logging/SentryEventJSException.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+import Sentry
+
+public protocol JSException {
+    var type: String { get }
+    var message: String { get }
+    var jsStacktrace: [StacktraceLine] { get }
+    var context: [String: Any] { get }
+    var tags: [String: String] { get }
+    var isHandled: Bool { get }
+    var handledBy: String { get }
+}
+
+public protocol StacktraceLine {
```

Comment:
Too bad we can't nest protocol declarations. What do you think giving it a more specialized name like `JSStacktraceLine`?

Reply:
…Unless you update to Xcode `15.3` and [Swift `5.10`](https://www.swift.org/blog/swift-5.10-released/)… which [just gained such ability via SE-404](https://github.com/apple/swift-evolution/blob/main/proposals/0404-nested-protocols.md) 😉

Reply:
It appears you still can't nest protocol in another protocol:

```
$ swift repl
Welcome to Apple Swift version 5.10 (swiftlang-5.10.0.13 clang-1500.3.9.4).
Type :help for assistance.
  1> protocol Foo { protocol Bar {} }
error: repl.swift:1:25: error: protocol 'Bar' cannot be nested in protocol 'Foo'
protocol Foo { protocol Bar {} }
                        ^


  1> struct Foo { protocol Bar {} }
  2>
```

Reply:
> Too bad we can't nest protocol declarations. What do you think giving it a more specialized name like `JSStacktraceLine`?

Sure, I'll rename it to make it more specific to JavaScript.

Reply:
> > Too bad we can't nest protocol declarations. What do you think giving it a more specialized name like `JSStacktraceLine`?
> 
> Sure, I'll rename it to make it more specific to JavaScript.

Suggestion applied in https://github.com/Automattic/Automattic-Tracks-iOS/pull/278/commits/d3694e8a352f8cac3f3c26f73125614408c01abc.